spec/lib/vmdb/loggers/container_logger_spec.rb

```diff
@@ -1,5 +1,5 @@
 RSpec.describe Vmdb::Loggers::ContainerLogger::Formatter do
-  it "stuff" do
+  it "logs a message" do
```

Comment:
I like the old description was better :trollface:

Reply:
It's not even "does stuff"... 🤣